### ISSUE TITLE
SetPlaybackRate ,the wave position is not synced.

### ISSUE THREAD

**Issue by netanelbit** [wavesurfer-js/wavesurfer.js#2349]
you can see it even in Stretcher example, play with the buttons (100, 75..) and see how the cursor "jumps" to incorrect position, I noticed that this behavior is from version 4.3 and above.

**Comment by sundayz** [comment]
Which backend are you using, and which browser. `setPlaybackRate` has different behaviour in different browsers

**Comment by netanelbit** [comment]
default backend - WebAudio
browser - chrome 86+

**Comment by sahi1422** [comment]
@sundayz , I also faced the same issue.

I would like to add that I also came up with the same workaround suggested by @netanelbit and it worked for me:
`pause , set the playback rate, and then resume play.`

Though, the audio at other playback rates than 1 sounded not right. Most likely the pitch of the audio gets changed while changing the playback Rate. @netanelbit , I ended up using `MediaElement` backend as it does not have these issues.

https://github.com/katspaugh/wavesurfer.js/issues/2197
